components/cardOurValues/cardOurValue.module.css

```diff
@@ -0,0 +1,42 @@
+.ourValueSubtitle {
+   text-transform: uppercase;
+   font-size: small;
+}
+.ourValueTitle {
+   font-size: xx-large;
+   padding-bottom: var(--spacing-40);
+}
+.ourValueCards {
+   display: flex;
+   gap: var(--spacing-20);
+   align-items: center;
+   justify-content: space-between;
+   flex-wrap: wrap;
+   
+}
+     
+.ourValueItem { 
+   border-bottom: 1px solid var(--border-color);
+   padding-bottom: var(--spacing-24);
+}
+
+/* For larger screens (desktops) */
+@media screen and (min-width: 768px) {
+   .ourValueItem {
+      flex: 0 1 48%;
+   }
+}
+
+/* For small screens (mobile devices) */
+@media screen and (max-width: 767px) {
```

Comment:
I love this! 😍 
Always good to think about mobile users!

Reply:
The way you can import the code is the following (if you are using mobile-first css)
```
.myCssClass {
  // Common styles
  margin: 16px;
  ...
  // mobile specific styles
  max-width: 500px;
}

@media screen and (min-width: 768px) {
   .myCssClass {
      // Here we overwrite the desktop specific styles
      max-width: 1600px;
   }
}
```